src/Http/Wolverine.Http/IHttpAware.cs

```diff
@@ -108,4 +108,37 @@ void IHttpAware.Apply(HttpContext context)
 public record CreationResponse<T>(string Url, T Value) : CreationResponse(Url)
 {
     
+}
+
+    
+#region sample_AcceptResponse
+
+/// <summary>
+/// Base class for resource types that denote some kind of request being accepted in the system.
+/// </summary>
+public record AcceptResponse(string Url) : IHttpAware
+{
+    public static void PopulateMetadata(MethodInfo method, EndpointBuilder builder)
+    {
+        builder.RemoveStatusCodeResponse(202);
```

Comment:
Is this supposed to remove 202?

`CreationResponse` removes 200, as that's one of the default status-codes wolverine adds.